src/unity/schemas/IdentityUser.yml

```diff
@@ -0,0 +1,34 @@
+properties:
+  idpeId:
```

Comment:
I think we should consider renaming this `id`, because that's the only ID there is from the client's perspective.

Reply:
I went ahead and renamed both of these to just `id` (`IdentityUser` and `IdentityOrganization`)